packages/generators/init-generator.ts

```diff
@@ -28,274 +19,212 @@ import { WebpackOptions } from "./types";
-		const { outputDir } = await Input(
-			self,
-			"outputDir",
-			"In which folder do you want to store your generated bundles?",
-			"'dist'",
-			this.autoGenerateConfig
-		);
-
-		this.usingDefaults = !outputDir || outputDir === "'dist'";
-
-		if (!this.usingDefaults) {
-			this.configuration.config.webpackOptions.output = {
-				chunkFilename: "'[name].[chunkhash].js'",
-				filename: "'[name].[chunkhash].js'"
-			};
-		}
-
-		const { langType } = await List(
-			self,
-			"langType",
-			"Will you use one of the below JS solutions?",
-			["No", LangType.ES6, LangType.Typescript],
-			"No",
-			this.autoGenerateConfig
-		);
-
-		langQuestionHandler(this, langType);
-		this.langType = langType;
-		if (this.langType !== "No") {
-			this.usingDefaults = false;
-		}
-		const { stylingType } = await List(
-			self,
-			"stylingType",
-			"Will you use one of the below CSS solutions?",
-			["No", StylingType.CSS, StylingType.SASS, StylingType.LESS, StylingType.PostCSS],
-			"No",
-			this.autoGenerateConfig
-		);
-		if (this.langType === "No") {
-			this.usingDefaults = true;
-		}
-		const { ExtractUseProps, regExpForStyles } = styleQuestionHandler(self, stylingType);
-		if (stylingType !== "No") {
-			this.usingDefaults = false;
-		}
-		// Ask if the user wants to use extractPlugin
-		const { useExtractPlugin } = await Input(
-			self,
-			"useExtractPlugin",
-			"If you want to bundle your CSS files, what will you name the bundle? (press enter to skip)",
-			"main",
-			this.autoGenerateConfig
-		);
-
-		if (regExpForStyles) {
-			const cssBundleName: string = useExtractPlugin;
-			this.dependencies.push("mini-css-extract-plugin");
-			this.configuration.config.topScope.push(
-				tooltip.cssPlugin(),
-				"const MiniCssExtractPlugin = require('mini-css-extract-plugin');",
-				"\n"
-			);
-			if (cssBundleName.length !== 0) {
-				(this.configuration.config.webpackOptions.plugins as string[]).push(
-					// TODO: use [contenthash] after it is supported
-					`new MiniCssExtractPlugin({ filename:'${cssBundleName}.[chunkhash].css' })`
-				);
-			} else {
-				(this.configuration.config.webpackOptions.plugins as string[]).push(
-					"new MiniCssExtractPlugin({ filename:'style.css' })"
-				);
-			}
-
-			ExtractUseProps.unshift({
-				loader: "MiniCssExtractPlugin.loader"
-			});
-
-			this.configuration.config.webpackOptions.module.rules.push({
-				test: regExpForStyles,
-				use: ExtractUseProps
-			});
-		}
-		if (this.usingDefaults) {
-			// Html webpack Plugin
-			this.dependencies.push("html-webpack-plugin");
-			const htmlWebpackDependency = "html-webpack-plugin";
-			const htmlwebpackPlugin = generatePluginName(htmlWebpackDependency);
-			(this.configuration.config.topScope as string[]).push(
-				`const ${htmlwebpackPlugin} = require('${htmlWebpackDependency}')`,
-				"\n",
-				tooltip.html()
-			);
-			(this.configuration.config.webpackOptions.plugins as string[]).push(`new ${htmlwebpackPlugin}({
+    public usingDefaults: boolean;
```

Comment:
Could you please point us where exactly you made some changes? It's difficult to understand

Reply:
Sure will do once I fix the conflicts, turns out my editor formatted these files on save taking into account the .editorconfig and .prettierrc which contains massive line length(233), would add better rules to prettier config in a separate PR.